create_keepalived_sysext.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
```

Comment:
```suggestion
export ARCH="${ARCH-x86-64}"
SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
```

Reply:
The other scripts have it declared on top because it's something the user can overwrite.